test/test_transformers.py

```diff
@@ -818,89 +832,6 @@ def test_bias_is_none(self):
         model(x, x, x)
         # completes without error
 
-    @parametrize("device", device_list)
```

Comment:
Why remove these tests?
Things that used to be errors/unexpected need to be updated but the bulk of the test still makes sense no? Or this is already done somewhere else?

Reply:
the removed checks were doing argument checks on combinations of inputs.  Removed them initially to not repeatedly have to change them as I explore solutions.  (github was the quickest way to run all tests, so I submitted PR revisions on every rev I wanted to test....)

In retrospect I wonder whether it's worthwhile to run all the parameter combination tests.

Reply:
Testing all combination of parameters is definitely the way to go to make sure we don't regress on corner cases.
In the grand scheme of things, we should have ModuleInfo testing for this that will allow us to do this easily but I think this is fine for now.

Adding back the updated tests here sounds good.